### PR TITLE
Added cluster level logging info

### PR DESCRIPTION
**Pull request by soumyalj**
@deniseschannon : Please review

### CODE REVIEW

File: content/rancher/v2.x/en/tasks/clusters/enabling-cluster-logging/_index.md [hunk 1]
```diff
@@ -3,4 +3,25 @@ title: Enabling Cluster Logging
 weight: 3600
 draft: true
 ---
-Coming Soon
+Cluster level logging allows us to obtain logs related to all containers in a cluster.
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/tasks/clusters/enabling-cluster-logging/_index.md]: What is a use case of why someone would want to collect logs? 

There is cluster level and project level logging. What is the difference between the two? Why would someone use cluster level versus project level logging?

File: content/rancher/v2.x/en/tasks/clusters/enabling-cluster-logging/_index.md [hunk 2]
```diff
@@ -3,4 +3,25 @@ title: Enabling Cluster Logging
...
+Once cluster level logging is enabled, the standard output and standard error for each container,
+log files are available under path `/var/log/containers/` on each host.
+
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/tasks/clusters/enabling-cluster-logging/_index.md]: When enabling logging, we add these logs onto each host as well as ship to any of the targetted logging solutions? Do these logs get rotated at all on the hosts? Do they eventually get purged?

File: content/rancher/v2.x/en/tasks/clusters/enabling-cluster-logging/_index.md [hunk 3]
```diff
@@ -3,4 +3,25 @@ title: Enabling Cluster Logging
...
+The log will be shipped to the any of the targets chosen. The targets are :
+
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/tasks/clusters/enabling-cluster-logging/_index.md]: Can more than 1 logging target be chosen or only one?

File: content/rancher/v2.x/en/tasks/clusters/enabling-cluster-logging/_index.md [hunk 4]
```diff
@@ -3,4 +3,25 @@ title: Enabling Cluster Logging
...
+ * Elasticsearch
+ * Splunk
+ * Kafka
+ * Syslog
+
+
+## Embedded elastic search
+
+1. Choose 'Embedded elastic search'
+2. Specify the CPU and Memory configuration
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/tasks/clusters/enabling-cluster-logging/_index.md]: What do these fields mean? What are you picking?

File: content/rancher/v2.x/en/tasks/clusters/enabling-cluster-logging/_index.md [hunk 5]
```diff
@@ -3,4 +3,25 @@ title: Enabling Cluster Logging
...
+ * Embedded elasticsearch
+ * Elasticsearch
+ * Splunk
+ * Kafka
+ * Syslog
+
+
+## Embedded elastic search
+
+1. Choose 'Embedded elastic search'
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/tasks/clusters/enabling-cluster-logging/_index.md]: Where do I find the cluster level logging? If this is the first step, then you've already assumed that we've already picked our cluster, navigated to tools->logging.

File: content/rancher/v2.x/en/tasks/clusters/enabling-cluster-logging/_index.md [hunk 6]
```diff
@@ -3,4 +3,25 @@ title: Enabling Cluster Logging
...
+2. Specify the CPU and Memory configuration
+3. In the prefix field (Index Patterns section), specify the cluster
+name for which we need to enable logging
+4. Add any custom log fields as required
+5. Click 'Save'
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/tasks/clusters/enabling-cluster-logging/_index.md]: What happens when you click save? What gets deployed in Rancher to start monitoring logging?

File: content/rancher/v2.x/en/tasks/clusters/enabling-cluster-logging/_index.md (hunk 3 above)
Comment (cloudnautique) [reply content/rancher/v2.x/en/tasks/clusters/enabling-cluster-logging/_index.md]: only one per cluster.
Only one per project.

File: content/rancher/v2.x/en/tasks/clusters/enabling-cluster-logging/_index.md (hunk 4 above)
Comment (cloudnautique) [reply content/rancher/v2.x/en/tasks/clusters/enabling-cluster-logging/_index.md]: Embedded elasticsearch is getting removed.

File: content/rancher/v2.x/en/tasks/clusters/enabling-cluster-logging/_index.md (hunk 6 above)
Comment (cloudnautique) [reply content/rancher/v2.x/en/tasks/clusters/enabling-cluster-logging/_index.md]: We will deploy fluentd daemonset into the cattle-logging namespace.

**Comment by MBishop17** [comment]
Closing in favor of #600

**Pull request closed without merging**